int8 COMMAND_NONE = 0
int8 COMMAND_CIRCLE = 1
int8 COMMAND_RTL = 2
int8 COMMAND_GOTO = 3

int8 command

float64 latitude
float64 longitude
